model/src/cfd.rs

```diff
@@ -296,6 +298,35 @@ impl Order {
             self.opening_fee,
         )
     }
+
+    const OUTDATED_AFTER_MINS: i64 = 10;
+
+    /// Defines when we consider the order to be sane within the system boundaries
+    ///
+    /// This is a safety net for the taker to prevent the taker from taking an outdated order.
+    pub fn is_safe_to_take(&self, now: OffsetDateTime) -> bool {
+        !self.is_creation_timestamp_outdated(now) && self.is_oracle_event_id_sane_to_take(now)
+    }
+
+    /// Check if the the maker's offer creation timestamp is outdated
+    ///
+    /// If the creation timestamp is older than 10 minutes the offer is considered outdated
```

Comment:
🔧 You never know when things might change... You could even argue that the docstring shouldn't need to explain what the code does (if it's clear enough).

```suggestion
    /// If the creation timestamp is older than `OUTDATED_AFTER_MINS` minutes the offer is considered outdated
```